x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -45067,7 +45067,7 @@
     "xpack.spaces.navControl.spacesMenu.changeCurrentSpaceTitle": "Modifier l'espace en cours",
     "xpack.spaces.navControl.spacesMenu.findSpacePlaceholder": "Rechercher un espace",
     "xpack.spaces.navControl.spacesMenu.noSpacesFoundTitle": "aucun espace trouvé",
-    "xpack.spaces.navControl.spacesMenu.selectSpacesTitle": "Vos espaces",
```

Comment:
@ek-so @elena-shostak unless our guidance has changed, we should not be editing translation files directly. Translations are handled by an external provider.